BMG-Stop-Plate/src/main.cpp

```diff
@@ -226,21 +226,23 @@
     oled.clear();    
     oled.setFont(SystemFont5x7);
     oled.set1X();
-    oled.print("  SHOT       TIME(s)");
+    oled.print("SHOT     TIME(s)");
     oled.println();
     oled.println();
-    oled.set2X();
+    //oled.set2X();
     oled.setFont(Arial_bold_14);
     oled.print(" " + hitNo);
+    oled.set2X();
 
     if (seconds.length() == 3) {
       oled.print(" ");
     } else {
-      oled.print("   ");
+      oled.print("  ");
     }
     oled.print(seconds);
-    oled.set1X();
-    oled.println("." + centiSecond);  
+    //oled.set1X();
```

Comment:
just delete this. we don't need to comment it out as we can see historical changes